x-pack/test/functional_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,281 @@
+    });
+
+    it('resolver Nodes navigation Right', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+      await (await testSubjects.find('east-button')).click();
+
+      const NodeData1 = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(NodeData1);
+      for (let i = 0; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+      }
+      await (await testSubjects.find('center-button')).click();
+    });
+
+    it('resolver Nodes navigation Center', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+      await (await testSubjects.find('east-button')).click();
+      await (await testSubjects.find('south-button')).click();
+
+      const NodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const NewNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(NodeData);
+      for (let i = 0; i < NewNodeDataStyle.length; i++) {
+        expect(parseFloat(NewNodeDataStyle[i].left)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(NewNodeDataStyle[i].top)).to.lessThan(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+      await (await testSubjects.find('center-button')).click();
+
+      const CenterNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const CenterNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(CenterNodeData);
+
+      for (let i = 0; i < CenterNodeDataStyle.length; i++) {
+        expect(parseFloat(CenterNodeDataStyle[i].left)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].left)
+        );
+        expect(parseFloat(CenterNodeDataStyle[i].top)).to.equal(
+          parseFloat(OriginalNodeDataStyle[i].top)
+        );
+      }
+    });
+
+    it('resolver Nodes navigation zoom in', async () => {
+      const OriginalNodeData = await pageObjects.endpointAlerts.getEndpointAlertResolverNodeData(
+        'resolverNode',
+        'style'
+      );
+      const OriginalNodeDataStyle = await pageObjects.endpointAlerts.parseStyles(OriginalNodeData);
+
+      await (await testSubjects.find('zoom-in')).click();
+      await browser.setWindowSize(2400, 1800);
```

Comment:
interesting that none of the other tests needed a specific browser size set, care to put a comment in as to how this is special, if it can help with potential future maintenance?

Reply:
I removed the browser
I was looking for the solution for the get all the nodes and this did not work